.golangci.yml

```diff
@@ -0,0 +1,16 @@
+run:
+  skip-dirs:
+    - pkg/client
+    - internal/pkg/cert
+    - internal/pkg/crv01
+
+linters:
+  enable:
+    - gofmt
```

Comment:
Although goimports does what gofmt does, but it doesn't inform is a file is not formatted properly. Enabling gofmt separately to have overall file formatting issue reported.